development/includes/_hero.pug

```diff
@@ -1,4 +1,4 @@
-div.hero-feature
+div.hero
```

Comment:
Please review your own pull request before tagging me. This is not the hover card, I don't want to see this change in this branch. Take this out then I'll review it again.